src/EditorFeatures/Text/Extensions.SnapshotSourceText.cs

```diff
@@ -267,9 +268,9 @@ public override SourceText WithChanges(IEnumerable<TextChange> changes)
             private class ChangedSourceText : SnapshotSourceText
             {
                 private readonly SnapshotSourceText _baseText;
-                private readonly ITextSnapshot _baseSnapshot;
+                private readonly ITextImage _baseSnapshot;
```

Comment:
No, it actually isn't.  We first clone _baseText.TextImage into a new TextBuffer, then we get teh .CurrentSnapshot of that TextBuffer and get the TextImage of that.  

Note: i can't tell why we would want to do that at all.  I can remove this field an djust have people go through _baseText if you want.